GeosCore/tomas_mod.F90

```diff
@@ -6355,16 +6507,17 @@ SUBROUTINE INIT_TOMAS( Input_Opt, State_Chm, State_Grid, RC )
     ! (bmy, 1/30/14)
     DATA_DIR = TRIM( Input_Opt%CHEM_INPUTS_DIR ) // 'TOMAS_201402/'
 
+    ! comment out if  shut off subgridcoag 14/12/23, bc
```

Comment:
Rather than commenting this out, I would recommend bracketing this code with `IF ( Input_Opt%SGCOAG ) THEN... ENDIF`. Or if the code is no longer needed, it can simply be removed.

Reply:
As commented above - would it be ok to keep with commented out for use as a basis for future developments that would be needed to restore function of SGCOAG? Currently the true option has the potential  for erroneous results due to resolution dependency issues.

Reply:
Yes, we will keep this as is then.